pkg/internal/cluster/logs/logs.go

```diff
@@ -130,6 +129,11 @@ func Collect(nodes []nodes.Node, dir string) error {
 // untar reads the tar file from r and writes it into dir.
 func untar(r io.Reader, dir string) (err error) {
 	tr := tar.NewReader(r)
+	defer func() {
```

Comment:
is this really needed? is is possible this func returns without error?

Reply:
The WIP is to prevent merge. This PR is just being used to obtain debugging information versus trying to trigger this failure mode locally because CI will run many jobs at once for me :-)

Reply:
(also that's why it checks if err != nil)